components/RepaymentInfo/RepaymentInfo.tsx

```diff
@@ -23,6 +51,21 @@ export function RepaymentInfo({ loan }: RepaymentInfoProps) {
         <dd>{longFormattedInterestRate}</dd>
         <dt>Payback at maturity</dt>
         <dd>{longFormattedEstimatedPaybackAtMaturity}</dd>
+        {!!loan.endDateTimestamp && (
+          <>
+            <dt>Maturity Date</dt>
+            <dd className={styles['maturity-date']}>
+              <div>
+                {maturityDate}
+                <Link href={calEventLink} passHref>
+                  <a target="_blank" rel="noreferrer">
```

Comment:
@adamgobes this should have an aria-label and a title, both could be something like "Add the end date of this loan to your Google Calendar"